src/views/OrderDetail.vue

```diff
@@ -354,7 +360,9 @@ export default defineComponent({
       this.store.dispatch('order/updatedOrderListItems', this.ordersList.items);
     },
     getGroupList (items: any) {
-      return Array.from(new Set(items.map((ele: any) => ele.parentProductId)))
+      if (items.length) {
```

Comment:
Why this is needed?

Reply:
In save method, after uploading the data we clear the state and navigate to purchase order page. The order detail component is not unmounted and as the state is empty it results in an error.

Reply:
I think if we are resetting items to [], we should not have error in any case and this handling is not required
Also, why not?
```
return [ ...new Set(items.map((ele: any) => ele.parentProductId))]
```